rclpy/rclpy/time_source.py

```diff
@@ -88,3 +108,14 @@ def clock_callback(self, msg):
         self._last_time_set = time_from_msg
         for clock in self._associated_clocks:
             clock.set_ros_time_override(time_from_msg)
+
+    def on_parameter_event(self, parameter_list):
+        for parameter in parameter_list:
+            if parameter.name == 'use_sim_time':
```

Comment:
nit: in this case we can break/return early.

Reply:
Fixed